Engine/src/Engine/SceneBuilder/Components.h

```diff
@@ -159,4 +160,20 @@ namespace Engine
 			destroyScript = [this] { delete m_instance; m_instance = nullptr; };
 		}
 	};
+
+	//A component used for collision detection in the physics system
+	struct PhysicsComponent
+	{
+		PhysicsComponent(glm::vec3& dimensions, glm::vec3& position)
+			: boundingBox(new AABB(dimensions, position)) {}
+		~PhysicsComponent()
+		{
+			if (boundingBox)
+			{
+				delete boundingBox;
+			}
+		}
+
+		AABB* boundingBox;
```

Comment:
would be nice to have a comment here with "Axis Aligned Bounding Box" so other people looking at this struct can get a better understanding (or at least an easier time googling). Just a suggestion

Reply:
I'd also think that some brief notes on how to use the component would be incredibly useful. I need to do that for the components I created as well 😅